JASP-Engine/JASP/R/ancova.R

```diff
@@ -1247,7 +1128,7 @@ Ancova <- function(jaspResults, dataset = NULL, options) {
     postHocTable$addColumnInfo(name="meanDiff",title=gettext("Mean Difference"), type="number")
     
     if (options$confidenceIntervalsPostHoc) {
-      thisOverTitle <- gettextf("%g%% CI for Mean Difference", options$confidenceIntervalIntervalPostHoc * 100)
+      thisOverTitle <- gettextf("% CI for Mean Difference", options$confidenceIntervalIntervalPostHoc * 100)
```

Comment:
```suggestion
      thisOverTitle <- gettextf("%s%% CI for Mean Difference", options$confidenceIntervalIntervalPostHoc * 100)
```
otherwise, `options$confidenceIntervalIntervalPostHoc * 100` won't be inserted into the string.

Reply:
Ah yea, haven't really worked with those gettext functions before...